edi_webservice_oca/tests/test_send.py

```diff
@@ -40,6 +40,20 @@ def _setup_records(cls):
                     endpoint: push/here
         """
         cls.record.type_id.set_settings(cls.settings1)
+        cls.a_user = cls.env["res.users"].create(
```

Comment:
```suggestion
        cls.a_user = cls.env["res.users"].with_context(no_reset_password=True).create(
```
or use `odoo.tests.common.new_test_user`